src/test/views/test_read_mipas.py

```diff
@@ -25,7 +25,7 @@ def test_read_mipas_file(mipas_basepath_pattern):
         basepath_pattern=str(mipas_basepath_pattern)
     )
     expect = {
-        'MJD': 54132.00168981482,
+        'MJD': 54132.00168981497,
```

Comment:
This could also explain the STW, mismatch, perhaps. There is some back and forth, where floating point differences could compound.

Reply:
well, this difference (1e-9) on mjd is about 90 microseconds...